models/access.go

```diff
@@ -124,13 +125,20 @@ func (u *User) GetRepositoryAccesses() (map[*Repository]AccessMode, error) {
 // GetAccessibleRepositories finds repositories which the user has access but does not own.
 // If limit is smaller than 1 means returns all found results.
 func (user *User) GetAccessibleRepositories(limit int) (repos []*Repository, _ error) {
-	sess := x.Where("owner_id !=? ", user.ID).Desc("updated_unix")
+	sess := x.Where("owner_id !=? ", user.ID).Desc("uid").Desc("updated_unix")
```

Comment:
@bkcsoft good point ! Besides, how does that have to do with "watch status" sorting in the title of this PR ?